project/profiles/capacity4more/modules/c4m/wysiwyg/c4m_wysiwyg_ckeditor/c4m_wysiwyg_ckeditor.module

```diff
@@ -104,3 +104,116 @@ function c4m_wysiwyg_ckeditor_media_wysiwyg_format_form_prepare_alter(
       'c4m_wysiwyg_ckeditor'
     ) . '/c4m_wysiwyg_ckeditor.js';
 }
+
+/**
+ * Define all external domains that use cookies.
+ *
+ * @return array
+ */
+function _c4m_wysiwyg_ckeditor_ckeditor_iframes_with_cookies() {
+  return [
+    'youtube',
+    'archive',
+    'dailymotion',
+    'google',
+    'slideshare',
+    'vimeo',
+    'ustream',
+    'prezi',
+    'issuu',
+    'scribd',
+    'tubechop',
+    'twitter',
+    'twitframe',
+    'mapbox',
+    'soundcloud',
+    'audioboom',
+    'mixcloud',
+    'mindmeister'
+  ];
+}
+
+/**
+ * Implements hook_filter_info().
+ */
+function c4m_wysiwyg_ckeditor_filter_info() {
+  $filters['c4m_wysiwyg_ckeditor_iframe_cck'] = array(
+    'title' => t('Iframe Cookie Consent Filter'),
+    'description' => t('Checks iframes and adds the EU Cookie Consent kit to it if applicable'),
+    'process callback' => '_c4m_wysiwyg_ckeditor_iframe_cck_filter_process',
+    'tips callback' => '_c4m_wysiwyg_ckeditor_iframe_cck_filter_tips',
+  );
+
+  return $filters;
+}
+
+/**
+ * Returns tips for defined filter.
+ *
+ * @param $filter
+ * @param $format
+ * @param bool $long
+ * @return string
```

Comment:
- Separate the @param and @return sections by a blank
- Description for the @return value is missing